habinezadalvan/LIG-app-api
# task-LUX-12-verify account

### Description

- create RESTful api to verify user account

- tests

### How to test
- clone repo

- checkout to **task-LUX-11-verify-email**

- run **yarn install**

- add all environment values to your .env file

- run **yarn start:dev** to start server

- run **yarn test** to run tests

- now you can go to **http://localhost:5050/graphql** to create user mutation

### How has this been tested?


### Jira
[LUX-12](https://amanilabs.atlassian.net/browse/LUX-12)
